pom.xml

```diff
@@ -3,9 +3,10 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.mesos</groupId>
-  <artifactId>storm</artifactId>
-  <version>0.9.6</version>
-  <name>storm</name>
+  <artifactId>storm-parent</artifactId>
+  <version>0.1.0</version>
```

Comment:
1. Yeah, we'll have to finagle the system to get the `build-release.sh` working with the release/assembly plugins of maven, won't work right out of the box.
2. I also don't have time, my zone is about to be called in the airport, going on vacation. ;-)
3. Yeah, let's just put -SNAPSHOT in for the moment.